src/models/campaigns.model.js

```diff
@@ -50,6 +51,8 @@ function createModel(app) {
     },
   );
 
+  campaign.index({ updatedAt: 1 });
```

Comment:
uses in this query
https://github.com/Giveth/feathers-giveth/blob/develop/src/services/donations/donations.hooks.js#L173